app.js

```diff
@@ -0,0 +1,37 @@
+let form = document.querySelector('form');
+
+form.addEventListener('submit', async function (event) {
+    event.preventDefault();
+    console.log(form.username.value);
+    let repoNames = await getGitHubRepo(form.username.value);
+    displayRepos(repoNames);
+})
+
+async function getGitHubRepo(username) {
+    try {
+        let resp = await fetch(`https://api.github.com/users/${username}/repos`);
+        let jsonResp = await resp.json();
+        if (resp.ok) {
+
+            let publicRepoNames = jsonResp
+                .filter(repo => repo.private === false)
+                .map(repo => repo.name);
+            return publicRepoNames;
+
+        } else {
+            console.log(jsonResp.message);
+        }
+    } catch (err) {
+        console.log('Something went wrong');
+    }
```

Comment:
Only console logs? Let's show some information to the user, otherwise how does he/she know what happened.